galaxy/values.yaml

```diff
@@ -118,16 +121,20 @@ persistence:
   size: 10Gi
   mountPath: /galaxy/server/database
 
-# extraVolumes:
+extraVolumes: []
 #   - name: shared-data
 #     persistentVolumeClaim:
 #       claimName: shared-data-pvc
 
-# extraVolumeMounts:
+extraVolumeMounts: []
 #   - name: shared-data
 #     mountPath: /mnt/project/shared-data
 
-# extraInitContainers:
+initContainers:
```

Comment:
Do we need to make this user configurable? My intuition here is that we don't and can just do this in the web handler always? Otherwise LGTM

Reply:
I don't mind making it not configurable, main reason was to be able to spin up a specific handler without requiring the other ones, and move it in case one wants to start with multiple replicas (since ideally the init container should be only once). I'm now thinking maybe going back to making it jobs is the way to go, that way it's independent of the handlers altogether?